benchmark/benchfeatures.cpp

```diff
@@ -98,10 +97,7 @@ struct option_struct {
         verbose = true;
         break;
       case 'a':
-        arch = parse_architecture(optarg);
-        if (arch == architecture::UNSUPPORTED) {
-          exit_usage(string("Unsupported option value -a ") + optarg + ": expected -a HASWELL, WESTMERE or ARM64");
-        }
+        simdjson::active_implementation = simdjson::available_implementations[optarg];
```

Comment:
That's safe with respect to any optarg?

Reply:
I think it returns nullptr if the parameter is invalid.

Reply:
Oh, you're right! I think I was assuming it would abort if it didn't exist, the way SIMDJSON_FORCE_IMPLEMENTATION does. I think I'll just remove the parameter; there's not much point to having it now that we have the variable.

Reply:
Parameter removed.